src/components/QuoteContainer.js

```diff
@@ -1,13 +1,21 @@
 import React, { useState } from 'react';
 import StartButton from './StartButton.js';
+import ItsKevButton from './ItsKevButton.js';
+import NotKevButton from './NotKevButton.js';
+
 import '../css/App.css';
 
 const QuoteContainer = () => {
-  const [quote, setQuote] = useState("Press the button for a quote!");  
+  const [currentQuote, setQuote] = useState("Press the button for a quote...");
+
   return (
         <div className="Quote-container">
-            <p className="Quote-text">{quote}</p>
-            <StartButton changeTheQuote={setQuote}/>
+            <div className="Quote-text-container">
+                <p className="Quote-text">{currentQuote}</p>
+            </div>
+            <StartButton currentQuoteProp={currentQuote} setQuoteProp={setQuote} />
```

Comment:
Another teeny point is that you don't necessarily need to call the prop a `Prop` by name. So `currentQuote={}` would suffice rather than `currentQuoteProp={}`

Reply:
Ah, I only did this to help me follow the flow of data so that I was clear what was going on!